test/tool/net/uuidv4_test.lua

```diff
@@ -0,0 +1,23 @@
+-- Copyright 2022 Justine Alexandra Roberts Tunney
```

Comment:
Copyright 2024?

Reply:
I just copied an existing one :)

Reply:
Yeah no problem there, I just usually update the copyright year whenever I'm making a new file

Reply:
fixed